fetchComponent.py

```diff
@@ -103,16 +142,20 @@ def buildPackageBoard(packageInfo):
     board["shape"] = [shape]
     return board
 
-def easyEdaToKicad(boardJson):
+def easyEdaToKicad(symbolJson, boardJson):
     """
     Convert board JSON, return pcbnew.BOARD
     """
     with TemporaryDirectory() as tmpDir:
-        easyFilename = os.path.join(tmpDir, "board.json")
+        symbolFilename = os.path.join(tmpDir, "symbol.json")
+        boardFilename = os.path.join(tmpDir, "board.json")
         kicadFilename = os.path.join(tmpDir, "board.kicad_pcb")
-        with open(easyFilename, "w") as easyFile:
+        with open(symbolFilename, "w") as schFile:
+            schFile.write(json.dumps(symbolJson, indent=2))
+        with open(boardFilename, "w") as easyFile:
             easyFile.write(json.dumps(boardJson, indent=4))
-        subprocess.check_call(["easyeda2kicad", easyFilename, kicadFilename])
+        subprocess.check_call(["easyeda2kicad", boardFilename, kicadFilename])
+        os.system("./LC2KiCad/build/lc2kicad -v " + symbolFilename + " -a ENL:1")
```

Comment:
Please, don't use `os.system` (especially, when the project already uses `subprocess.*`) - you need to only execute a program; not execute it from the shell. It makes no difference here, but it is good to be consistent with the project and also, do not invoke a shell if not needed (as a shell can perform, e.g., expansion. Also, users use various shells with inconsistent behavior).

Also, based on my proposal above; I think it makes sense to invoke LC2KiCAD from PATH, not from a directory.

Reply:
OK, thanks for the explanation of the difference.

Reply:
Then, the right way for calling the subprocess is this one?
`subprocess.check_call(["lc2kicad", "-v", "symbolFilename", "-a", "ENL:1"])`

Reply:
Yes.